vhost/test_add_hdr.py

```diff
@@ -178,7 +180,7 @@ def config_append_directive(self, hdrs, location=None):
 
     def make_chain(self, hdrs):
         orig_hdrs = [('X-My-Hdr', 'original text'),
-                     ('X-My-Hdr-2', 'other original text')]
+                     ('X-My-Hdr-2', '0123456789' * N)]
```

Comment:
It seems there is no need for the tests, see the review of https://github.com/tempesta-tech/tempesta/pull/1497 .